presto-main/src/main/java/com/facebook/presto/server/QueryStateInfo.java

```diff
@@ -63,34 +68,38 @@ private QueryStateInfo(
         this.user = requireNonNull(user, "user is null");
         this.catalog = requireNonNull(catalog, "catalog is null");
         this.schema = requireNonNull(schema, "schema is null");
-        requireNonNull(resourceGroupChain, "resourceGroupChainInfo is null");
```

Comment:
Keep this check so you get a good error message (since all the others have them)